src/addresses.rs

```diff
@@ -0,0 +1,120 @@
+use mimir::rubber::Rubber;
+use mimir::Poi;
+use mimirsbrunn::admin_geofinder::AdminGeoFinder;
+use mimirsbrunn::labels::format_addr_name_and_label;
+use mimirsbrunn::labels::format_street_label;
+use mimirsbrunn::utils::find_country_codes;
+use std::ops::Deref;
+use std::sync::Arc;
+
+/// Read the osm address tags and build a mimir address from them
+///
+/// For the moment we read mostly `addr:city` or `addr:country`
+/// if available we also read `addr:postcode`
```

Comment:
Sentence seems a bit weird. Please add some punctuation.

Reply:
I updated and move this comment to the `find_address` function that is responsible for reading Poi properties and build the address.